guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -1,32 +1,32 @@
 [id="Installing_the_OpenSCAP_Plugin_{context}"]
 = Installing the OpenSCAP Plug-in
 
-You can install and enable the OpenSCAP plug-in to generate OpenSCAP compliance reports.
-The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP smart proxy plug-in, and the OpenSCAP Hammer CLI plug-in.
+You can install and enable the OpenSCAP plug-in to generate SCAP compliance reports.
+The OpenSCAP plug-in consists of the OpenSCAP {Project} plug-in, the OpenSCAP Hammer CLI plug-in, the OpenSCAP {SmartProxy} plug-in, and modules for policy deployment.
 
 .Procedure
 . Install the OpenSCAP plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
+# {foreman-installer} \
+--enable-foreman-plugin-openscap \
+--enable-foreman-cli-openscap \
+--enable-foreman-proxy-plugin-openscap
 ----
-ifndef::satellite[]
-. Optional: Install the OpenSCAP Hammer CLI plug-in on your {ProjectServer}:
 +
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-installer} --enable-foreman-cli-openscap
-----
-endif::[]
+Append one of the following arguments to the command:
 
-. Install the OpenSCAP plug-in on any {SmartProxyServers}:
+* `--foreman-proxy-plugin-openscap-ansible-module true` for Ansible deployment of compliance policies
+* `--foreman-proxy-plugin-openscap-puppet-module true` for Puppet deployment of compliance policies
+. Optional: Install the OpenSCAP plug-in on any {SmartProxyServers}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
+# {foreman-installer} --enable-foreman-proxy-plugin-openscap
 ----
-. In the {ProjectWebUI}, navigate to *Configure > Puppet Classes*.
-. Click *Import environments from {foreman-example-com}*.
 +
-You can use Puppet to install and configure the OpenSCAP plug-in on your {ProjectServer} and {SmartProxies}.
+Append one of the following arguments to the command:
+
+* `--foreman-proxy-plugin-openscap-ansible-module true` for Ansible deployment of compliance policies
+* `--foreman-proxy-plugin-openscap-puppet-module true` for Puppet deployment of compliance policies
```

Comment:
Similar, `capsule` pulls in `satellite-common` which in turn pulls in `ansiblerole-foreman_scap_client`, but not the Puppet side. Sorry to make this part way more complex.

Reply:
Again, this doc module doesn't apply to satellite.